src/PresentationalComponents/Dashboard/Dashboard.js

```diff
@@ -32,7 +32,7 @@ const Dashboard = () => {
                     {intl.formatMessage(messages.dashboardTitle)}
                 </Title>
             </PageSection>
-            <Main className='ins-l-dashboard'>
+            <Main className={ `ins-l-dashboard ${ permission.advisor !== 'true' && ' ins-m-no-subscription ' }` }>
```

Comment:
wait do we want this to be checking advisor's permission here?

Reply:
I had it there because the modifier class needs to exist at the top level of the dashboard because it adjusts the the layout. Is there a downside to checking there?

Reply:
Or a suggestion for how to do it a different way

Reply:
I mean doing the modifier on the class itself is fine, I was just curious why you were checking `permission.advisor` and not `permission.subscriptions`

Reply:
oh yepppp thanks for catching that!

Reply:
updated